cluster/examples/kubernetes/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml

```diff
@@ -108,6 +108,25 @@ spec:
             - name: keys-tmp-dir
               mountPath: /tmp/csi/keys
         {{ end }}
+        {{ if .EnableVolumeReplicationSideCar }}
```

Comment:
@Madhu-1 We would also need these changes in the csi repo, right?

Reply:
Yes, this is needed in the cephcsi repo. Will work on cephcsi later